WootricSDK/WootricSDK/WTRiPADSocialShareView.m

```diff
@@ -28,6 +28,7 @@
 #import "NSString+FontAwesome.h"
 #import "WTRiPADThankYouButton.h"
 #import "SimpleConstraints.h"
+#import <WootricSDK/WootricSDK-Swift.h>
```

Comment:
Same here, why do we need this import?

Reply:
When we are using swift code in Objective C then we have to import swift header file like above.